3.7/ubuntu/Dockerfile

```diff
@@ -0,0 +1,271 @@
+# The official Canonical Ubuntu Bionic image is ideal from a security perspective,
+# especially for the enterprises that we, the RabbitMQ team, have to deal with
+FROM ubuntu:18.04 AS release
+
+# Start a new builder image so that the release image remains unmodified
+FROM release AS builder
+
+# Install curl & CA certificates
+# Verify that CA certificates work correctly - https://github.com HTTP request succeeds
+# I believe in progressive checks: run the simplest checks to ensure that the installed dependency works
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends curl ca-certificates && \
+    curl --version && \
+    curl --verbose --head --fail --fail-early --silent https://github.com
+
+# Install dependencies required to build Erlang/OTP from source
+# http://erlang.org/doc/installation_guide/INSTALL.html
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends make && \
+    make --version
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends gcc && \
+    gcc --version
+
+# Required to configure Erlang/OTP before compiling
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends autoconf && \
+    autoconf --version
+
+# Required to set up host & build type when compiling Erlang/OTP
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends dpkg-dev && \
+    dpkg-architecture
+
+# Required for Erlang/OTP HiPE support
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends m4 && \
+    m4 --version
+
+# Required for Erlang/OTP new shell & observer_cli - https://github.com/zhongwencool/observer_cli
+# It would be nice to have a basic check here, same as we do for other dependencies, but I'm not sure what/how to check this...
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends libncurses5-dev
+
+# Required to verify OpenSSL & RabbitMQ artefacts
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends gnupg && \
+    gpg --version
+
+# Sometimes keys fail to return from PGP keyservers
+# Chose a PGP keyserver that uses multiple TLDs and has the highest SRV score in Europe, North America & Oceania - https://sks-keyservers.net/status/
+ARG PGP_KEYSERVER=pgpkeys.eu
```

Comment:
See https://github.com/docker-library/official-images/issues/4252, especially https://github.com/docker-library/official-images/issues/4252#issuecomment-405749749 / https://github.com/docker-library/php/pull/666 / https://github.com/docker-library/rabbitmq/pull/270 (which has pretty much eliminated PGP flakiness issues for us both in Travis builds and on the official-images build servers, so I'd appreciate if we stuck to one of the servers that https://github.com/tianon/pgp-happy-eyeballs/blob/master/hack-my-builds.sh hijacks for the sake of the official images server / Travis builds).